docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3'
+
+services:
+  app:
+    build: .
+    image: electic-io
+    container_name: electric-io
+    command: npm run start
+    ports:
+      - 3000:3000
+    env_file:
+      - .env
+    environment:
```

Comment:
This allows the environment override for simulating. I could expand this for any of the .env options if we need to. I figure people can modify their own .env files for the other settings and not check them in if they are contributing, but toggling simulating seems like something you'd want to do more often.